include/openspace/scene/profile.h

```diff
@@ -121,7 +121,7 @@ class Profile {
      * and all of the property & asset changes that were made since startup.
      */
     void saveCurrentSettingsToProfile(const properties::PropertyOwner& rootOwner,
-        const std::string& currentTime,
+        std::string currentTime, 
```

Comment:
Did you intend to change this to a std::string_view?

Reply:
No, it can be a `std::string` because it's value is saved in the `Time t` parameter.  The const reference before would have needed a copy in some situations which can be prevented now:

A (before):
```
std::string t = "timestring";
profile.saveCurrentSettingsToProfile(owner, t);   // passed by reference

//
t.time = currentTime; /// < Makes a copy of the string
```

B (now):
```
std::string t = "timestring";
profile.saveCurrentSettingsToProfile(owner, std::move(t));   // moving into the parameter, not copying

//
t.time = std::move(currentTime); /// < Moving and also not making a copy
```